reana_db/models.py

```diff
@@ -575,43 +575,56 @@ def can_transition_to(self, next_status):
         current_transition = (self.status, next_status)
         return current_transition in ALLOWED_WORKFLOW_STATUS_TRANSITIONS
 
+    def update_workflow_timestamp(self, new_status):
```

Comment:
Out of curiosity, do the changes if this function get commited automatically? Wondering if we need a `Session.commit()` here.

Reply:
Added. It worked because there is an eventual commit in: https://github.com/reanahub/reana-db/blob/6c5ebc2a69b54818652957e6867a66f022eccf04/reana_db/models.py#L548-L571

After updating the status.